site/en/tutorials/images/classification.ipynb

```diff
@@ -152,7 +152,7 @@
       "source": [
         "import pathlib\n",
         "dataset_url = \"https://storage.googleapis.com/download.tensorflow.org/example_images/flower_photos.tgz\"\n",
-        "data_dir = tf.keras.utils.get_file('flower_photos', origin=dataset_url, untar=True)\n",
+        "data_dir = tf.keras.utils.get_file('flower_photos.tar', origin=dataset_url, extract=True)\n",
         "data_dir = pathlib.Path(data_dir)"
```

Comment:
This change leaves `data_dir` set to `/path/to/flower_photos.tar`, which is the tarball, not the data directory.

Suggestion: change line 156 to `data_dir = pathlib.Path(data_dir).with_suffix('')`, then `next(data_dir.glob('roses/*.jpg'))` will work.

I realise our notebook tests are not fully working at the moment, but it'd be great if you could test your changes before sending a PR.

Reply:
```suggestion
        "data_dir = pathlib.Path(data_dir).with_suffix('')"
```